lib/osf-components/addon/components/draft-registration-card/template.hbs

```diff
@@ -5,96 +5,104 @@
 >
     <h4 local-class='DraftRegistratrionCard__title' data-test-draft-registration-card-title>
         {{#if this.draftRegistration}}
-            {{this.draftRegistration.registrationSchema.name}}
+            <OsfLink
+                data-analytics-name='view_registration'
+                @route='registries.drafts.draft'
+                @models={{array this.draftRegistration.id}}
+            >
+                {{this.draftRegistration.registrationSchema.name}}
+            </OsfLink>
```

Comment:
What were the reasons behind making this a link? I think it (the fact that the text in the link does not refer to where the link takes the user) a bit confusing for the general user and misleading in terms of accessibility. What I mean is the link takes you to the draft registration detail page not a registration schema detail page.

Reply:
That's completely valid. I think at the time we were trying to minimize the buttons in that section and suggested we remove the edit button and make the title a link instead. Conversations with product led to us keeping the buttons but also making the title a link as well.

Reply:
Ohhhh -- now that users can edit titles, we should probably put the actual title here instead of the schema name. Schema name might make sense as like a subtitle sort of thing, or as another field down in the card body.

Reply:
Oh right, I forgot about that. I think that would make a lot more sense (and help people know which registration is which).